src/mds/SessionMap.h

```diff
@@ -54,6 +55,9 @@ enum {
  */
 
 class Session : public RefCountedObject {
+  using clock = coarse_mono_clock;
+  using time = coarse_mono_time;
+
```

Comment:
There's now a redeclaration which breaks the build: https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=xenial,DIST=xenial,MACHINE_SIZE=huge/14179//consoleFull